test/unit/app/models/concerns/default_lead_image_test.rb

```diff
@@ -0,0 +1,32 @@
+require "test_helper"
+
+class DefaultLeadImageTest < ActiveSupport::TestCase
+  setup do
+    @img_one = create(:featured_image_data, file: File.open(Rails.root.join("test/fixtures/minister-of-funk.960x640.jpg")))
+    @img_two = create(:featured_image_data, file: File.open(Rails.root.join("test/fixtures/big-cheese.960x640.jpg")))
+    @img_three = create(:featured_image_data, file: File.open(Rails.root.join("test/fixtures/images/960x640_gif.gif")))
+    @lead_organisation_with_image = create(:organisation, default_news_image: @img_one)
+    @lead_organisation_with_no_image = create(:organisation, default_news_image: nil)
+    @supporting_organisation_with_image = create(:organisation, default_news_image: @img_two)
+    @worldwide_organisation_with_image = create(:published_worldwide_organisation, default_news_image: @img_three)
+  end
+
+  test "it returns the default news image from the first lead organisation if present" do
+    edition = build(:standard_edition, lead_organisations: [@lead_organisation_with_image])
+    assert_equal @img_one, edition.default_lead_image
+  end
+
+  test "it returns the default news image from a supporting organisation if the first lead does not have an image" do
+    edition = build(:standard_edition,
+                    lead_organisations: [@lead_organisation_with_no_image],
+                    organisations: [@supporting_organisation_with_image, @lead_organisation_with_no_image])
+    assert_equal @img_two, edition.default_lead_image
+  end
+
+  test "it returns the default news image from the worldwide organisations if organisations are missing" do
+    ConfigurableDocumentType.setup_test_types(build_configurable_document_type("test_type_with_images_and_attachments", {}))
+    edition = create(:standard_edition, configurable_document_type: "test_type_with_images_and_attachments", organisations: [])
+    edition.edition_worldwide_organisations.create([{ document: @worldwide_organisation_with_image.document }])
+    assert_equal @img_three, edition.default_lead_image
+  end
```

Comment:
Thanks for adding these tests - super useful! 👏 

Looking at the [code](https://github.com/alphagov/whitehall/blob/46bfc04e03b7e1e730ec7ce267ebaec12b432fd8/app/models/concerns/standard_edition/default_lead_image.rb#L7-L9), we're still missing a couple of tests:

1. Test for if the lead org and the first supporting organisation lack a lead image, but the second supporting organisation has a lead image. (I think this comes out as `nil`)
2. Test for if the lead org and its supporting orgs lack a lead image, and the first published worldwide org lacks a lead image, but the second published worldwide org has a lead image. (Again, comes out as `nil` I think)